test/onnx/expect/TestOperators.test_view_flatten.expect

```diff
@@ -39,16 +121,16 @@ graph {
     }
   }
   output {
-    name: "6"
+    name: "11"
     type {
       tensor_type {
         elem_type: 1
         shape {
           dim {
-            dim_value: 1
+            dim_param: "Reshape11_dim_0"
           }
           dim {
-            dim_value: 24
+            dim_param: "Reshape11_dim_1"
```

Comment:
Note this dynamic change is introduced by `tensor.numel` being traced dynamically, and will be reverted in the next PR in the stack where constant folding for `onnx::ReduceProd` is introduced.